2-mandatory/1-weather-report.js

```diff
@@ -12,14 +12,19 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    const statements = [];
+    cities.forEach(city => {
```

Comment:
I believe this exercise was meant to be solved by simple iteration as forEach is only introduced during week 4. On the other hand if you are already using items from week 4 then prefer solving this exercise using `map`: https://syllabus.codeyourfuture.io/js-core-1/week-4/lesson#map